src/Microsoft.ML.Transforms/ExtensionsCatalog.cs

```diff
@@ -33,11 +33,12 @@ public static class ExtensionsCatalog
             => new MissingValueIndicatorEstimator(CatalogUtils.GetEnvironment(catalog), outputColumnName, inputColumnName);
 
         /// <summary>
-        /// Creates a new output column, of boolean type, with the same number of slots as the input column. The value in the output column
-        /// is true if the value in the input column is missing.
+        /// Create a <see cref="MissingValueIndicatorEstimator"/>, which copies the data from the column specified in <see cref="InputOutputColumnPair.InputColumnName" />
+        /// to a new column: <see cref="InputOutputColumnPair.OutputColumnName" />.
```

Comment:
>/// to a new column: <see cref="InputOutputColumnPair.OutputColumnName" />. [](start = 8, length = 75)

@natke does the colon ':' look good to you?

Reply:
Looks fine to me @shmoradims